src/go/util/xds_name.go

```diff
@@ -67,6 +67,9 @@ const (
 	// The service control server cluster name.
 	ServiceControlClusterName = "service-control-cluster"
 
+	// The ADS cluster name.
+	AdsClusterName = "ads-cluster"
```

Comment:
Where is this used?  I thought we are using Pipe name as the cluster name

Reply:
Thanks, forgot to remove this. Originally was going to hardcode the pipe name, decided to make it a flag.